packages/client/src/containers/LandingPage/LandingPage.Style.css

```diff
@@ -1,3 +1,3 @@
 .landing-page-container {
-  color: #687808;
+  color: #ffffff;
```

Comment:
@rebeccahockley  Hi. Everything looks good. But we need to change color format to RGB.